README.md

```diff
@@ -84,12 +116,15 @@ For RVM to operate properly you need to have certain resources deployed in your
 
 - RVM main roles in RVM account[^2]
 - An IAM OIDC provider in both RVM and target accounts[^3]
+- A verified email and domain with [Amazon Simple Email Service](https://docs.aws.amazon.com/ses/latest/dg/Welcome.html) [^4]
```

Comment:
From our Slack conversation, sounds like a verified domain is no longer needed

Reply:
Verified domain is not required for testing, but in production when different requesters with different emails request access, you need to verify the domain instead of each email address.

Reply:
Weird. Wouldn't it make more sense to create an email dedicated to this (`breakglass@company.lab`) and register that in SES?